content/guides/public-path.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Public Path
+sort: 14
+contributors:
+  - rafaelrinaldi
+---
+
+[webpack-public-path]: http://webpack.github.io/docs/configuration.html#output-publicpath
+
+# `publicPath`
```

Comment:
You can drop this header. That way it's consistent with the rest of the content.